akka-projection-grpc/src/test/scala/akka/projection/grpc/replication/ReplicationSettingsSpec.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.grpc.replication
+
+import akka.actor.testkit.typed.scaladsl.ActorTestKit
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.projection.grpc.replication.scaladsl.ReplicationProjectionProvider
+import akka.projection.grpc.replication.scaladsl.ReplicationSettings
+import akka.projection.grpc.replication.javadsl.{ ReplicationSettings => JReplicationSettings }
+import akka.projection.grpc.replication.javadsl.{ ReplicationProjectionProvider => JReplicationProjectionProvider }
+import com.typesafe.config.ConfigFactory
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.concurrent.duration.DurationInt
+
+class ReplicationSettingsSpec extends AnyWordSpec with Matchers {
+
+  trait MyCommand
+
+  "The ReplicationSettings" should {
+    "Parse from config" in {
+      implicit val system: ActorSystem[Unit] = ActorSystem[Unit](
+        Behaviors.empty[Unit],
+        "parse-test",
+        ConfigFactory.parseString("""
+         // #config
+         my-replicated-entity {
+           # which of the replicas this node belongs to, should be the same
+           # across the nodes of each replica Akka cluster.
+           self-replica-id = dca
```

Comment:
include an environment variable override to illustrate that the same config can be deployed on all replicas, but overridden (from Kubernetes deployment)